src/webgpu/api/validation/state/device_lost/destroy.spec.ts

```diff
@@ -146,6 +146,93 @@ Tests creating buffers on destroyed device. Tests valid combinations of:
     }, awaitLost);
   });
 
+g.test('mapping,mappedAtCreation')
+  .desc(
+    `
+Tests behavior of mappedAtCreation buffers when destroying the device (multiple times).
+- Various usages
+- Wait for .lost or not
+  `
+  )
+  .params(u =>
+    u
+      .combine('usage', [
+        GPUConst.BufferUsage.MAP_READ,
+        GPUConst.BufferUsage.MAP_WRITE,
+        GPUConst.BufferUsage.COPY_SRC,
+      ])
+      .combine('awaitLost', [true, false])
+  )
+  .fn(async t => {
+    const { awaitLost, usage } = t.params;
+
+    const b1 = t.createBufferTracked({ size: 16, usage, mappedAtCreation: true });
+    t.expect(b1.mapState === 'mapped', 'b1 before destroy 1');
+
+    await t.executeAfterDestroy(() => {
```

Comment:
We are still pending this decision right?

Reply:
Yes.